src/initialData.ts

```diff
@@ -2,32 +2,64 @@ const initialData: InitialData = {
   tasks: {
     "task-1": {
       id: "task-1",
-      content: "Take out the garbage",
+      content: "Hover over ☰ next to Task App on nav bar to add new column",
```

Comment:
I would say "at the top left of the screen" instead of "next to Task App"

Reply:
done